Build broken

Trying to build under Haiku hrev57701, 64bit, I get these sort of failures:

```
                 from common/src/gui/scrollview/ScrollView.cpp:3:
/boot/system/develop/tools/lib/gcc/x86_64-unknown-haiku/13.2.0/include/c++/pstl/glue_algorithm_defs.h:15:10: fatal error: execution_defs.h: No such file or directory
   15 | #include "execution_defs.h"

                 from libwarpsharp/src/complex.cpp:9:
/boot/system/develop/tools/lib/gcc/x86_64-unknown-haiku/13.2.0/include/c++/bits/basic_string.h:51:11: fatal error: cow_string.h: No such file or directory
   51 | # include "cow_string.h"
```

I couldn't check on beta4 yet...

--- a/build/OverriddenJamRules
+++ b/build/OverriddenJamRules
@@ -33,9 +33,17 @@ actions Chmod1
 # headers
 
 if $(OSPLAT) = X86 {
-	HDRS_INCLUDES_SEPARATOR ?= -I- ;
-	HDRS_LOCAL_INCLUDES_OPTION ?= -I ;
-	HDRS_SYSTEM_INCLUDES_OPTION ?= -I ;
+	if $(TARGET_PLATFORM) = haiku64 {
+		HDRS_INCLUDES_SEPARATOR ?= "" ;
+		HDRS_LOCAL_INCLUDES_OPTION ?= "-iquote " ;
+		HDRS_SYSTEM_INCLUDES_OPTION ?= "-isystem " ;
+
+		rule FIncludes { return "-iquote $(<)" ; }
+	} else {
+		HDRS_INCLUDES_SEPARATOR ?= -I- ;
+		HDRS_LOCAL_INCLUDES_OPTION ?= -I ;
+		HDRS_SYSTEM_INCLUDES_OPTION ?= -I ;
+	}
 } else {
 	HDRS_INCLUDES_SEPARATOR ?= -i- ;
 	HDRS_LOCAL_INCLUDES_OPTION ?= -I ;

